.github/workflows/platform-executables.yml

```diff
@@ -0,0 +1,56 @@
+name: PlatformExecutable Release
+on:
+  workflow_dispatch:
+    inputs:
+      formula:
+        description: 'Artifact formula name'
```

Comment:
[Q] What's this input param for? Doesn't seem quite intuitive to me...could we add more to description? Any use-case when we might need a different artifact name?

Reply:
This is the prefix of artifact we are releasing in Github releases. Currently it is `twilio-{version}.{extension}`, by default it will take `twilio` as prefix, but if we want give any other prefix we can provide it as input. 
The name `formula` I took it from Librarian. If we want to change it we can rename it as `Artifact Prefix` or any suggestions?